objcgen/Makefile

```diff
@@ -15,3 +15,29 @@ ifeq ($(SHELLCHECK),)
 else
 	$(SHELLCHECK) $^
 endif
+
+VERSION = 0.1
+FRAMEWORK_DIR = _build/Library/Frameworks/Xamarin.Embeddinator-4000
+INSTALLER_FILES = Xamarin.Embeddinator-4000-$(VERSION).pkg
+
+package:: bin/Debug/objcgen.exe
+	mkdir -p $(FRAMEWORK_DIR)/Commands/
+	mkdir -p $(FRAMEWORK_DIR)/Versions/$(VERSION)/bin
+	ln -fs $(VERSION) $(FRAMEWORK_DIR)/Versions/Current
+	cp bin/Debug/objcgen.exe $(FRAMEWORK_DIR)/Versions/Current/bin/
+	cp bin/Debug/Mono.Options.dll $(FRAMEWORK_DIR)/Versions/Current/bin/
```

Comment:
we should copy the .pdb files (or .mdb for Mono.Options.dll) to get more useful bug reports